### DISCUSSION TITLE
Should we bring back on:Hover video preview

### DISCUSSION THREAD

**Discussion by akshaynarisetti** [JdeRobot/RoboticsAcademy#1684] [Ideas]
Should we bring back on hover video preview on the cards on the main page of RoboticsAcademy Exercises?

After refactoring the code to the current one this feature seems to be missed out.

This will be a plus to the user experience :)

solves #1674

**Comment by jmplaza** [comment]
yes, sure. Go ahead @akshaynarisetti :-)
